default.nix

```diff
@@ -0,0 +1,7 @@
+{ pkgs ? import <nixpkgs> { } }:
+pkgs.rustPlatform.buildRustPackage {
+  pname = "tabry-rs";
+  version = "0.1";
```

Comment:
Is that supposed to match the version in Cargo.toml? Not that at matters at this point, just wondering. When I bump versions I have to change both right? I'll add a comment about that in the Cargo.toml, or you can

Reply:
Oh, good question... I'm not sure, actually

Reply:
It looks like it's included in the hash for the dependencies, so it makes sense to update it lock & step with `Cargo.toml`, I'll add a comment there